source/training/index.md

```diff
@@ -0,0 +1,62 @@
+# <i icon-name="book"></i> Training Materials
+
+```{toctree}
+:hidden:
+
+kickoff
+session1
+session2
+session3
+session4
+session5
+session6
+session7
+session8
+session9
+```
+
+The following training materials were originally developed with funding under the EU Govtech4all program.
```

Comment:
```suggestion
The following training materials were developed with co-funding from the French government’s DINUM and the European Commission under the [GovTech4All](https://interoperable-europe.ec.europa.eu/collection/govtechconnect/govtech4all) program.
```

+ please add the “Co-funded by the European Union” [logo](https://ec.europa.eu/regional_policy/information-sources/logo-download-center_en) and the [DINUM logo](https://fr.wikipedia.org/wiki/Fichier:Logo_DINUM_2020_RVB.png)

Reply:
Please check https://github.com/openfisca/openfisca-doc/pull/316/commits/1915f2746d5b31462835484c04b7ce4df2bba750